src/components/Signup/Signup.test.js

```diff
@@ -138,4 +144,26 @@ describe('Signup', () => {
       expect(container.querySelector('#iti-item-io').nextElementSibling.id).toBe('iti-item-bn'),
     );
   });
+
+  it('should render flag based in ipinfoClient result', async () => {
+    // Arrange
+    const dependencies = defaultDependencies;
+    const location = { search: 'test', pathname: '/signup' };
+
+    // Act
+    const { container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <DopplerIntlProvider locale="en">
+          <Router>
+            <Signup location={location} />
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    await waitFor(() =>
+      expect(container.querySelector('.iti__selected-flag').title).toBe('Puerto Rico: +1'),
```

Comment:
how do you know show `puerto rico : +1` ? maybe it's better add `ipinfoclient` in the test, what do you think?

Reply:
I didn't find any other attr to check that the shown country is the correct. What do you mean with add ipinfoclient?

Reply:
Perhaps you can have some variable, with a key and a country description to avoid misunderstandings like this:
```
const selectedCountry = {
  key: 'PR',
  description: 'Puerto Rico: +1'
};
```
and then
```
expect(container.querySelector('.iti__selected-flag').title).toBe(selectedCountry.description),
```

Reply:
maybe it's better put the string in the expect, we use that way in most of the tests

Reply:
que tal usar tablas? https://jestjs.io/docs/en/api#2--describeeachtablename-fn-timeout-

```js
it.each`
  countryCode | expectedDescription
  ${'PR'}     | ${'Puerto Rico: +1'}
  ${'ES'}     | ${'España: +34'}
`('should render selected country based in ipinfoClient result', ({countryCode, expectedDescription}) => {
    // Arrange
    const dependencies = defaultDependencies;
    // . . .
```

Reply:
I don't realize how can I use the countryCode in the hardcoded dependency ipinfoClient that returns the async response. I think that this won't work in this case.

Reply:
oh, and override dependencies does not work :(

Reply:
Exactly, so I think that by the moment I will use strings in the expects to be consistent with other test files as @fcoronelmakingsense said.